src/components/pages/RoleSignup/Applications/Styles/mentorApplication.css

```diff
@@ -1,57 +1,29 @@
-.mentorApplication{
+.container{
   display: flex;
-  padding:2vh 6vw;
   justify-content: center;
 }
 
 .mentorTitle{
-  border-bottom:solid 1px rgba(128, 128, 128, 0.342); 
-  background-color:#003D71;
-  text-align:center;
-  color:#FFF !important;
-  padding:1vh;
-  
+  background-color: #003D71;
+  color: #ffffff !important;
+  text-align: center;
+  width: 100%;
+  border-top-right-radius: 12px;
+  border-top-left-radius: 12px;
+  padding: 2rem;
 }
 
-.applicationForm{
-  border-radius:'4px';
-  border:solid 1px rgba(128, 128, 128, 0.342);
-  display: flex;
-  
-}
+.col-container{
+  padding: 2rem;
 
-.locationUS{
-  display: flex;
-  flex-direction: column;
 }
-#mentorSubmitButton{
-  background-color: #003D71;
-  color: #FFF;
-  margin-bottom: 2vh;
+.em{
```

Comment:
please put a space between the .em and "{".

Reply:
this has been resolved